_posts/0000-01-02-mwagoner85.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![17-coding-memes-for-the-frustrated-software-engineer](https://user-images.githubusercontent.com/70118294/90999431-2e1c2500-e58c-11ea-8083-a9d13c66e24f.jpg)